ImperatorToCk2/Output.java

```diff
@@ -0,0 +1,737 @@
+        } else {
+            out.println (tab+"250.1.1 ={");    
+        }
+        out.println (tab+tab+"death= yes");
+        out.println (tab+"}");
+        out.println ("}");
+
+      
+        out.flush();
+        fileOut.close();
+
+        return irKING;
+    }
+
+    public static String dynastyCreation(String name, String id, String Directory) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+        String tab = "	";
+
+        Directory = Directory + VM + "common" + VM + "dynasties";
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "c_" + id + ".txt");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+
+        String date1 = "100.1.1";
+        String date2 = "1066.9.15";
+
+      
+        out.println (id+"=");
+        out.println ("{");
+        out.println (tab+"name="+VMq+name+VMq);
+        out.println (tab+"used_for_random=no");
+        out.println ("}");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+
+    public static String btitleCreation(String name, String Directory, int id) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+        String tab = "	";
+        String oldName = name;
+        name = name.toLowerCase();
+        name = name.replace(' ','_');
+
+        if (id == 103) { //Leon in Brittany and Spain have the same name in definition.csv
+            name = "french_leon";  
+        }
+
+        Directory = Directory + VM + "common" + VM + "landed_titles";
+
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "b_" + name + ".txt");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+
+      
+        out.println ("c_"+name+"={");
+        out.println (tab+"b_"+name+"={");
+        out.println (tab+"}");
+        out.println ("}");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+
+    public static String localizationCreation(String[] name, String title, String Directory) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+
+        ArrayList<String> oldFile = new ArrayList<String>();
+
+        oldFile = Importer.importModLocalisation(Directory);
+
+        Directory = Directory + VM + "localisation";
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+      
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "converted_title_localisation.csv");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+        int aqq = 0;
+
+        try {
+
+            while (flag == 0) {
```

Comment:
It looks like you're just copying the original file with no modifications. Use java.nio.Files.copy to copy it, and delete the Importer.importModLocalisation function